src/components/UserInfo/UserInfo.scss

```diff
@@ -1 +1,3 @@
-//
```

Comment:
Add a brief comment at the top of the file describing the purpose of the styles.